src/client/marble-manager.js

```diff
@@ -125,9 +126,16 @@ const makeTextSprite = function(message, options = {}) {
 	let texture = new THREE.Texture(canvas);
 	texture.needsUpdate = true;
 
-	let spriteMaterial = new THREE.SpriteMaterial({ map: texture });
+	let spriteMaterial = new THREE.SpriteMaterial({ 
+		map: texture,
+		sizeAttenuation: false,
+		depthWrite: false,
+		depthTest: false,
+	});
+
 	let sprite = new THREE.Sprite(spriteMaterial);
-	sprite.scale.set(4, 1, 1.0);
+	sprite.scale.set(0.65, 0.25, 1.0);
```

Comment:
This results in a stretched looking name. It's also a tad big as it obfuscates a lot of the marbles and scenery. It should be large enough to be legible, but not much bigger than that.

It would be nice to have the size configurable, but that is out of the scope of this PR / issue.